src/status_im/ui/screens/wallet/add_new/views.cljs

```diff
@@ -120,7 +120,8 @@
             status        [:keycard/pin-status]
             error-label   [:keycard/pin-error-label]
             retry-counter [:keycard/retry-counter]]
-    [react/keyboard-avoiding-view {:style {:flex 1}}
+    [react/keyboard-avoiding-view {:style {:flex 1}
+                                   :ignore-offset true}
```

Comment:
[SELF NOTE] Do we ever use `:ignore-offset false`?

Reply:
false is default